cluster/service_test.go

```diff
@@ -170,6 +171,84 @@ func Test_NewServiceSetGetNodeAPIAddrTLS(t *testing.T) {
 	}
 }
 
+func Test_NewServiceTestExecuteQueryAuthNoCredentials(t *testing.T) {
+	ml := mustNewMockTransport()
+	db := mustNewMockDatabase()
+
+	// Test that for a cluster with no credential store configed
+	// all users are authed for both operations
+	var c CredentialStore = nil
+	t.Log(c)
```

Comment:
Why are you logging credentials during testing? Not a big deal, just wondering if you have a specific reason.

Reply:
Just a holdover from some debugging.  Will clean up.